examples/platform/nrf528xx/util/streamer/streamer_nrf5.cpp

```diff
@@ -0,0 +1,107 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      Source implementation of an input / output stream for stdio targets.
+ */
+
+#include "shell.h"
+
+#ifdef NRF_SHELL_STREAMER
+
+#include "app_uart.h"
+#include "bsp.h"
+#include "nrf_uarte.h"
+
+namespace chip {
+namespace Shell {
+
+const size_t c_rx_buffer_size = 256; /**< UART RX buffer size. */
+const size_t c_tx_buffer_size = 512; /**< UART TX buffer size. */
+
+void uart_error_handle(app_uart_evt_t * p_event)
+{
+    // if (p_event->evt_type == APP_UART_COMMUNICATION_ERROR)
```

Comment:
could we replace commented-out code with documentation-style comments on what should be done here if desired?

Reply:
Agree, thanks